style/values/specified/image.rs

```diff
@@ -43,10 +43,9 @@ fn gradient_color_interpolation_method_enabled() -> bool {
 pub type Image = generic::Image<Gradient, SpecifiedUrl, Color, Percentage, Resolution>;
 
 // Images should remain small, see https://github.com/servo/servo/pull/18430
-#[cfg(feature = "gecko")]
+#[cfg(any(feature = "gecko", feature = "servo"))]
```

Comment:
You can just remove the cfg since there is no other possibility than servo and gecko.

Reply:
Done.